pkg/kubelet/cm/memorymanager/memory_manager.go

```diff
@@ -222,6 +226,7 @@ func (m *manager) RemoveContainer(containerID string) error {
 	// if error appears it means container entry already does not exist under the container map
 	podUID, containerName, err := m.containerMap.GetContainerRef(containerID)
 	if err != nil {
+		klog.Errorf("[memorymanager] Failed to get container %s from container map error: %v", containerID, err)
```

Comment:
Error or Warning or Info?
Can be `RemoveContainer` called twice?
Can be `RemoveContainer` called before `AddContainer`?

Reply:
Looks like it can, I do not sure what the default threading model for the kubelet, does it run in the single thread or does it use a number of threads.

See for example this question - https://kubernetes.slack.com/archives/C0BP8PW9G/p1587419000430000

I can change it to the warning, I think it will be more appropriate, WDT?

Reply:
Yeah, I would also pick up warning.

Reply:
done